spec/spec_helper.rb

```diff
@@ -169,7 +169,7 @@ module Shell
   config.filter_run_excluding :aes_256_gcm_only => true unless aes_256_gcm?
   config.filter_run_excluding :broken => true
   config.filter_run_excluding :not_wpar => true unless wpar?
-  config.filter_run_excluding :not_fips => true unless fips?
+  config.filter_run_excluding :not_fips => true if fips?
```

Comment:
what is not_fips exactly? a test we want to exclude if we are not running under fips? it might be easier to make sure this isn't backwards if we named it better.

```
config.filter_run_excluding :with_fips_only => true unless fips?
config.filter_run_excluding :fips_enabled => true unless fips?
config.filter_run_excluding :fips_disabled => true if fips?
config.filter_run_excluding :without_fips_only => true if fips?
config.filter_run_excluding :not_supported_under_fips => true if fips?
```